### ISSUE TITLE
NAS paramter understanding

### ISSUE THREAD

**Issue by hdjsjyl** [TuSimple/simpledet#129]
Thanks for your good work. 
In the config file of retina_nasfpn, the iteration is the following:
end_epoch = 25
lr_iter = [15272 * 15 * 16 // (len(KvstoreParam.gpus) * KvstoreParam.batch_image),
                       15272 * 20 * 16 // (len(KvstoreParam.gpus) * KvstoreParam.batch_image)]

What does the "15272" mean? If I want to train it on our own dataset, how to change this parameter? Do you have advice about the epoch numbers? Any advice will be appreciated, thanks.

**Comment by xchani** [comment]
15272 seems to be the number of images in training, thereby the learning rate will be changed by 15 epoch and 20 epoch. For your own dataset, replace it by number of images in training dataset.

**Comment by johnlanbor** [comment]
@xchani  accoding to nas-fpn , train data is coco2017 train, so the number of images semms not 15272
